subset/network/test_ntpupdat_send

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
+
+REPORT=/tmp/report.txt
+
+ping -n -c 3 $TARGET_IP
+
+cp /scans/monitor.pcap capture.pcap
+
+python network_tests.py network.ntp.update capture.pcap $TARGET_IP
```

Comment:
As mentioned previously, the three test_ files can all just be one, and then call this python network_tests.py three times, rather than three separate test files.  The separation into the "three tests" comes when there's three different RESULT lines, but there should only be one Docker container and one test_ file.

Reply:
Because the tests are not meant to be run at the same time. We need extra monitor.pcap files for different phases. The monitor.pcap mechanism is not sufficient for purpose.